sonar-plugin-api/src/test/java/org/sonar/api/server/ws/RequestTest.java

```diff
@@ -49,6 +49,11 @@ public String method() {
     }
 
     @Override
+    public String getMediaType() {
+      return "application/json";
```

Comment:
```MimeTypes``` is in sonar-server currently. As soon as all refactorings/improvements are done on WsTester or WsActionTester, I'd like all these classes to be moved in sonar-plugin-api. It will allow to use MimeTypes.